client/implementation/src/main/java/io/smallrye/graphql/client/typesafe/impl/reflection/ParameterInfo.java

```diff
@@ -104,9 +106,12 @@ public Object getValue() {
     public String getName() {
         if (parameter.isAnnotationPresent(Name.class))
             return parameter.getAnnotation(Name.class).value();
+        JsonbProperty jsonbProperty = parameter.getAnnotation(JsonbProperty.class);
```

Comment:
Can you not use Jandex here ? Then we do not need the optional dependency on Jakarta json bind

Reply:
Yeah, I already thought about that. But that would only make sense if _all_ reflection would be done via Jandex. I want to do that anyways, so I can fully use Power-Annotations, but that would mean a larger rewrite.

Should I remove the dependency and thereby the support for JSON-B for now? We currently don't need it in our project.

Reply:
Maybe we just add a ticket so that we remember to do this later ? Also, maybe change to provided ? And maybe handle classnotfound ? I am not sure to be honest. Maybe we can discuss this afternoon ?

Reply:
Filed the Jandex topic as #693